etc/kayobe/environments/ci-multinode/README.md

```diff
@@ -0,0 +1,48 @@
+# Multinode Test Environment 
+
+## Set up hosts
+1. Create four baremetal instances with a centos 8 stream LVM image, and a 
+Centos 8 stream vm
+2. SSH into each baremetal and run `sudo chown -R centos:.` in the home directory,
+   then add the lines 
+   >`10.0.0.34 pelican pelican.service.compute.sms-lab.cloud`  
+   >`10.205.3.187 pulp-server pulp-server.internal.sms-cloud`  
+
+   to /etc/hosts (if you're waiting on them starting up, you can progress until 
+   `kayobe overcloud host configure` without this step)
+
+## Basic Kayobe Setup
+1. SSH into the VM
+1. `sudo dnf install -y python3-virtualenv`
+1. `mkdir src` and `cd src`
+1. Clone https://github.com/stackhpc/stackhpc-kayobe-config.git, then checkout
+   commit f31df6256f1b1fea99c84547d44f06c4cb74b161 
```

Comment:
I suppose this is to avoid picking up the Ceph changes? It should probably point to the `multinode-env` branch here though